DESCRIPTION

```diff
@@ -2,24 +2,20 @@ Package: shiny.router
 Type: Package
 Title: Basic Routing for Shiny Web Applications
 Version: 0.2.3
-Authors@R: c(person("Filip", "Stachura",
-                    email = "filip@appsilon.com",
-                    role = c("aut")),
-            person("Dominik", "Krzemiński",
-                    email = "dominik@appsilon.com",
-                    role = c("aut")),
-            person("Krystian", "Igras",
-                    email = "krystian@appsilon.com",
-                    role = c("aut")),
-            person("Developers", "Appsilon",
-                    email = "support+opensource@appsilon.com",
-                    role = c("cre", "aut")),
-              person("Servet Ahmet", "Çizmeli", email = "ahmet.cizmeli@contractors.appsilon.com", role = c("ctb")),
-            person(family = "Appsilon", role = c("cph"))
-           )
+Authors@R:
+  c(
+    person("Ryszard", "Szymański", email = "opensource+ryszard@appsilon.com", role = c("cre", "aut")),
+    person("Filip", "Stachura", email = "filip@appsilon.com", role = c("aut")),
+    person("Dominik", "Krzemiński", email = "raymon92@gmail.com", role = c("aut")),
+    person("Krystian", "Igras", email = "krystian8207@gmail.com", role = c("aut")),
+    person("Servet Ahmet", "Çizmeli", email = "ahmet.cizmeli@contractors.appsilon.com", role = c("ctb")),
+    person("Appsilon Sp. z o.o.", role = "cph", email = "opensource@appsilon.com")
```

Comment:
Is it `opensource@appsilon.com` or `support+opensource@appsilon.com`? I see [that in shiny.fluent we use a different email](https://github.com/Appsilon/shiny.fluent/blob/master/DESCRIPTION#L8)

Reply:
We used support email for some time, but now the opensource@appsilon.com one should be used. Some packages are still not updated.